popUp.js

```diff
@@ -109,4 +111,17 @@ form.addEventListener('submit', (e) => {
     e.preventDefault();
     errorMsg.innerText = msg.join(', ');
   }
+
+  const formData = {
+    storeName: localStorage.setItem('nameValue', nameValue),
+    storeEmail: localStorage.setItem('emailValue', emailValid),
```

Comment:
good job by saving data in local storage but formData object is should be used as one object wraping all form inputs and store them as one variable in localStorage this will reduce times you call localStorage update your object with name and email and message then setItem in localStorage once 
const formData = {
    name: nameInput.value,
    email: emailInput.value,
    message: messageInput.value,
};

Reply:
Sorry, your instructions are not clear to me. Can you please explain a little easy and clearly I didn't get it.